### ISSUE TITLE
Claims file requests - need more people on the team who can do data prep

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#5533]
## Description 
As a member of the mobile team, I want to be able to send new claims file requests (in staging) to test users, so that I can design, develop or test the mobile app. In particular, we use 1-2 file requests every release during RC testing, so we inevitably whittle down our open requests and need to refresh them periodically - more often if we're actually doing active work in the claims feature area. 

As part of this story we want to:   
- Get more people on the mobile team set up to do claims file request data prep just Narin and Therese

Unfortunately, unlike secure messaging (sister ticket #5532) there is no set process to get this access, that I know of. Patrick Saxton and/or Narin Ratana had worked with Paul Shute to get the original access in Feb 2022, so @narin may know how we could best reach out this time around to get the ball rolling.

## Notes & Open Questions
Once folks have access to the underlying system (big shrug on how that will happen), an instructional video for "how to do the prep" [already exists from the original training back in Feb 2022](https://drive.google.com/file/d/10_DJKi23ZMz3gDNoWU8Gcrg8If81Nces/view) (system UI has had some minor changes but not too bad)

## Ticket Checklist

- [x] Accounts have been created
- [x] Account for @theodur confirmed
- [x] Account for @DJUltraTom confirmed
- [x] Account for @rbontrager confirmed
- [x] Account for @timwright12 confirmed

**Comment by bischoffa** [comment]
Been in progress since June. Moving to blocked.

**Comment by timwright12** [comment]
@bischoffa @TKDickson @narin @theodur @rbontrager @DJUltraTom Do we have any updates or action items on this one? Do we need to reach out to Paul?

**Comment by TKDickson** [comment]
@timwright12 I have no idea the status of this ticket. The last I heard about it -- which was prior to my maternity leave -- was that I had gotten everyone the information that was necessary for folks to make progress on this while I was gone. 

Has any progress been made in the last 9 months, and if so, what? I do not know.

**Comment by TKDickson** [comment]
@bischoffa my comment made me sound primarily responsible/like I had work to do, let me try again 😆 

The next step - as Tim stated - is for someone to reach out to Paul Schute to request access for X people (this is not a formal process, but it's how a group of people got access the first time). Those people will need to have working PIV cards & access to the VA network. IDK if anyone ever did that outreach, and why not if they didn't.

My responsibility, before leaving on FMLA, was to make sure that people would know how to use the access once they had it. I've got the video with the original walkthrough linked above, and I also did a targeted demo about file access with some people additionally (I want to say Narin, Rachael, Greta, Holly and Stacy? There were people on the meeting walkthrough because they were working in claims, not because they were going to get access). That was the end of my stuff for this ticket, and it all happened 9 months ago.

**Comment by narin** [comment]
I had reached out to Paul via Slack in September and then sent an email a week later. Did not receive a reply to either. Maybe someone else will have better luck or we can get PO support.

**Comment by timwright12** [comment]
I can pick this up from here

**Comment by timwright12** [comment]
Having some account issues, hopefully fixed soon. Then I'll be able to start this.

**Comment by TKDickson** [comment]
@timwright12 - Been thinking about this ticket again this week since @rbontrager has needed to ping me a couple times already; H&B is working on a ticket that requires a lot of claims file submissions. How's progress?

**Comment by timwright12** [comment]
I have a PIV appointment on March 12th, should be smooth after that

**Comment by timwright12** [comment]
Messaged Paul today in MS Teams to request access for Theo, Tom, Rachael, and myself

**Comment by timwright12** [comment]
Accounts have been created for @theodur @rbontrager @theodur and myself.

Can you all confirm access is working for yourself:

> Once folks have access to the underlying system (big shrug on how that will happen), an instructional video for "how to do the prep" [already exists from the original training back in Feb 2022](https://drive.google.com/file/d/10_DJKi23ZMz3gDNoWU8Gcrg8If81Nces/view) (system UI has had some minor changes but not too bad)

**Comment by theodur** [comment]
Confirmed I was able to sign in ✅

**Comment by rbontrager** [comment]
I'm also able to sign in.

**Comment by timwright12** [comment]
@DJUltraTom can you try this out now that you have a PIV card?

**Comment by DJUltraTom** [comment]
running into issues accessing the VBMS site through CAG, when attempting to sign into the VBMS site through SSO PIV CARd, browser hangs .

**Comment by timwright12** [comment]
ok, please keep trying, if it still doesn't work you might have to get on a call with OIT

**Comment by bischoffa** [comment]
Moving this to QA epic for better tracking and updating the sprints

**Comment by bischoffa** [comment]
@timwright12 @DJUltraTom is there anything to escalate? This ticket has been opened for over a year.

**Comment by DJUltraTom** [comment]
Attempted to verify my account last week and ran into issues with my card reader being recognized, after a forced OS update it is being recognized again, but I am in the group of people falling into an issue opening Citrix desktop (reports exist in DSVA slack as well) will attempt again early next week<img width="1739" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNEVnQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b8ee10e17b690894e34689bd925dac73d1bed8d3/Screenshot%202024-05-29%20at%202.46.42%E2%80%AFPM.png" alt="Screenshot 2024-05-29 at 2.46.42 PM.png" />

**Comment by DJUltraTom** [comment]
I am able to access CAG and the VA intranet, but I cannot find information on where to access the VBMS portal, the video training doesn't have info on how to get to it and when I enter the URL in the video I get a blank page (not loading)
@TKDickson can you provide guidance?

**Comment by TKDickson** [comment]
@DJUltraTom the video starts after the login sequence/once you're in VBMS, yeah. There was a slack thread (when everyone else got & confirmed their access) that mentioned the URLs you need for authentication, and they're also in the [how stuff works QA doc](https://docs.google.com/document/d/1nqL9Wx4MDHju9mzDWvUWvqCBhE9r9Ef5U8TtoiCe1-A/edit#heading=h.7c6b2ybbwi8b) in 'claims'. Although if you were using the URL in the video, that's not too far off, so..... hopefully this works for you 🤞

**Comment by DJUltraTom** [comment]
WooHoo! verified I can log in!
@timwright12 if you are able to log. inas well we can close this ticket
